src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/dto/RequestDto.java

```diff
@@ -46,6 +51,30 @@ public class RequestDto extends MessageDto {
     @NotNull
     private DT_RequestStateEnum state;
 
+    /**
+     * Technical identifier for a {@link org.eclipse.tractusx.puris.backend.common.api.domain.model.Message}.
+     * <p>
+     * Only set for existing entities.
+     */
+    @JsonIgnore
+    private UUID uuid;
+
+    /**
+     * Steering information of a {@link Request} or {@link Response} api message.
+     */
+    @NotNull
+    @JsonProperty("headers")
+    private MessageHeaderDto header;
+
+    /**
+     * List of actual content of the payload.
+     * <p>
+     * May contain also errors.
+     */
+    @NotNull
+    @JsonProperty("payload")
+    private List<ProductStockRequestForMaterialDto> payload = new ArrayList<>();
+
     /**
```

Comment:
Duplicated Javadoc. Do we still need MessageContentDto, if we're directly using the ProductStockRequestForMaterialDto?

Reply:
Not sure... your call, I'd say.

Reply:
Then, plese just remove the javadoc.